packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatBackedCallAdapter.ts

```diff
@@ -116,12 +116,16 @@ export class CallWithChatBackedCallAdapter implements CallAdapter {
     await this.callWithChatAdapter.resumeCall();
   };
   /* @conditional-compile-remove(PSTN-calls) */
-  public addParticipant = async (
-    participant: CommunicationIdentifier,
+  public async addParticipant(participant: PhoneNumberIdentifier, options?: AddPhoneNumberOptions): Promise<void>;
+  /* @conditional-compile-remove(PSTN-calls) */
+  public async addParticipant(participant: CommunicationUserIdentifier): Promise<void>;
+  /* @conditional-compile-remove(PSTN-calls) */
+  public async addParticipant(
+    participant: PhoneNumberIdentifier | CommunicationUserIdentifier,
     options?: AddPhoneNumberOptions
-  ): Promise<void> => {
-    await this.callWithChatAdapter.addParticipant(participant, options);
-  };
+  ): Promise<void> {
+    return await this.callWithChatAdapter.addParticipant(participant, options);
```

Comment:
nitty nit: You can just `return this.callWithChatAdapter.addParticipant(...)`.

No need to `await` the Promise here since you don't do anything with the result. Caller can `await` when the want.